javascript/ql/src/Expressions/MissingAwait.qhelp

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>
+In JavaScript, <code>async</code> functions always return a promise object.
+To obtain the underlying value of the promise, the <code>await</code. operator or a call to <code>then</code. should be used.
+Attempting to use a Promise object instead of its underlying value can lead to expected behavior.
```

Comment:
```suggestion
Attempting to use a Promise object instead of its underlying value can lead to unexpected behavior.
```

Also, you capitalize `Promise` here, but not in the previous two lines.  
I think both are fine, but you should be consistent.

Reply:
Good point - I'm gonna stick to the uncapitalized version in promise.